src/controllers/order/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Body, Tags, Security, ValidateError, SuccessResponse, Response, Controller, Post, Route } from 'tsoa'
+import { Logger } from 'pino'
+
+import { DemandRequest, DemandResponse } from '../../models'
+import { logger } from '../../lib/logger'
+import { BadRequest, NotFound } from '../../lib/error-handler'
+import Database from '../../lib/db'
+import { getMemberByAddress, getMemberBySelf } from '../../lib/services/identity'
+
+const SUBTYPE = 'order'
+const STATE = 'created'
+
+@Route('order')
+@Tags('order')
+@Security('bearerAuth')
+export class order extends Controller {
+  log: Logger
+  db: Database
+  constructor() {
+    super()
+    this.log = logger.child({ controller: '/order' })
+    this.db = new Database()
+  }
+
+  /**
+   * A Member creates a new demand for a order by referencing an uploaded parameters file.
+   * @summary Create a new order demand
+   * @param parametersAttachmentId The attachment's identifier
+   */
+  @Post()
+  @Response<BadRequest>(400, 'Request was invalid')
+  @Response<NotFound>(404, 'Demand was not found')
+  @Response<ValidateError>(422, 'Validation Failed')
+  @SuccessResponse('201')
+  public async createCapacity(@Body() { parametersAttachmentId }: DemandRequest): Promise<DemandResponse> {
```

Comment:
function name

Reply:
thanks! indeed, just blindly thought that since the file name is already an entity we can call it `create`, my bad, it has been updated.